release-content/0.14/release-notes/13579_State_Identity_Transitions.md

```diff
@@ -0,0 +1,7 @@
+Identity state transitions are a niche feature that's been missing in previous implementations.
```

Comment:
```suggestion
Users have sometimes asked for us to trigger exit and entry steps when moving from a state to itself.
While this has its uses (refreshing is the core idea), it can be surprising and unwanted in other cases.
We've found a compromise that lets users hook into this type of transition if it's something they need.
```